jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -335,7 +336,9 @@
                   (- (/ (m->mm (send bin-box :dimensions :x)) 2.0))
                   0.0 (/ (m->mm (send bin-box :dimensions :z)) 2.0))
                 offset) :world)
-      (send bin-coords :rotate (* pi (/ coords-pitch 180)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 1)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 2)) :z)
+      (send bin-coords :rotate (deg2rad (aref rpy 0)) :x)
```

Comment:
Also, rotating order in local is `yaw-pitch-roll`:
```
40.irteusgl$ setq b (make-coords)
#<coordinates #X50fe5b8  0.0 0.0 0.0 / 0.0 0.0 0.0>
41.irteusgl$ send b :scale 6
#<coordinates #X50fe5b8  0.0 0.0 0.0 / 0.0 0.0 0.0>
42.irteusgl$ objects (list a b c)
(#<body #X493f8e0 (:cube 100.0 100.0 100.0) 0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X50fe5b8  0.0 0.0 0.0 / 0.0 0.0 0.0> #<coordinates #X5105708  0.0 0.0 0.0 / 1.0 1.0 1.0>)
43.irteusgl$ send b :rotate 1 :z :local
#<coordinates #X50fe5b8  0.0 0.0 0.0 / 1.0 0.0 0.0>
44.irteusgl$ send b :rotate 1 :y :local
#<coordinates #X50fe5b8  0.0 0.0 0.0 / 1.0 1.0 0.0>
45.irteusgl$ send b :rotate 1 :x :local
#<coordinates #X50fe5b8  0.0 0.0 0.0 / 1.0 1.0 1.0>
```
![screenshot from 2017-05-20 11 56 51](https://cloud.githubusercontent.com/assets/14994939/26272458/70834c6e-3d53-11e7-9032-2d91ddc5e43b.png)